demo/top_down_img_demo.py

```diff
@@ -66,13 +68,15 @@ def main():
             bbox = ann['bbox']
             person_bboxes.append(bbox)
 
-        # test a single image, with a list of bboxes.
-        pose_results, heatmaps = inference_top_down_pose_model(
+        # test a single image, with a list of bboxes
+        pose_results, returned_outputs = inference_top_down_pose_model(
             pose_model,
-            image_name,
+            image,
```

Comment:
Maybe we should use `image_name` here. Right?

Reply:
That's right. Fixed.